tests/test_configurations/test_loader.py

```diff
@@ -52,6 +52,13 @@ def test_windows_uri_path():
         assert output == f"file:/{'/'.join(path.parts)}/"
 
 
+@pytest.mark.parametrize("aea_config_yaml", Path(ROOT_DIR).rglob("**/aea-config.yaml"))
+def test_load_aea_config(aea_config_yaml):
+    """Test loading aea-config.yaml file"""
+    config_loader = ConfigLoader("aea-config_schema.json", AgentConfig)
+    config_loader.load(open(aea_config_yaml))
```

Comment:
We should have a negative test case as well

Reply:
the negative, as in empty, is covered in tests as well